### PR TITLE
Docs (markdown files): Fix typos discovered by codespell

### PR DESCRIPTION
**Pull request by cclauss**
Follow-up to #12308

[`codespell -w **/*.md`](https://pypi.org/project/codespell/)

### CODE REVIEW

File: tests/manual/abbreviation-level-2.md [hunk 1]
```diff
@@ -311,7 +311,7 @@ Let's write a basic `_createFormView()` function, just to create an instance of
 
 We also need to create a function, which will give us the target position for our balloon from user's selection. We need to convert selected view range into DOM range. We can use {@link module:engine/view/domconverter~DomConverter#viewRangeToDom `viewRangeToDom()` method} to do so.
 
-Finally, let's add our balloon amd fpr, view to the `init()` method.
+Finally, let's add our balloon and fpr, view to the `init()` method.
```
Comment (cclauss) [inline tests/manual/abbreviation-level-2.md:314]: Should `fpr,` be `form`?  On qwerty, that would be one-key-to the-right on both o->p and m->,

Comment (mlewand) [reply tests/manual/abbreviation-level-2.md]: I'm pretty sure it should be form. We can look at the inline comment in the code snippet itself:

> // Create the balloon and the form view.

So

> Finally, let's add our balloon amd fpr, view to the \`init()\` method.

Should become:

> Finally, let's add our balloon and form view to the \`init()\` method.

**Comment by godai78** [comment]
Great job, thanks!

**Review by godai78 — APPROVED** [review]
LGTM

**Pull request merged**